<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BadgeSettings</class>
 <widget class="QWidget" name="BadgeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbxOffsetV">
     <property name="minimum">
      <number>-40</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblOffsetV">
     <property name="text">
      <string>Vertical offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2" colspan="3">
    <widget class="MaterialBadge" name="badge">
     <property name="text">
      <string>3</string>
     </property>
     <property name="relativePosition">
      <pointf>
       <x>18.000000000000000</x>
       <y>18.000000000000000</y>
      </pointf>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbxType">
     <item>
      <property name="text">
       <string>Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Icon</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbxOffsetH">
     <property name="minimum">
      <number>-40</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOffsetH">
     <property name="text">
      <string>Horizontal offset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="MaterialAvatar" name="avatar">
     <property name="pixmap">
      <pixmap resource="resources/resources.qrc">:/images/sikh.jpg</pixmap>
     </property>
     <property name="size">
      <number>60</number>
     </property>
     <property name="image" stdset="0">
      <pixmap resource="resources/resources.qrc">:/images/sikh.jpg</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkDisabled">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialAvatar</class>
   <extends>QWidget</extends>
   <header location="global">QtMaterialWidgets/avatar.hpp</header>
  </customwidget>
  <customwidget>
   <class>MaterialBadge</class>
   <extends>QWidget</extends>
   <header location="global">QtMaterialWidgets/badge.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
